torch/utils/hipify/cuda_to_hip_mappings.py

```diff
@@ -7253,6 +7254,47 @@
             "cublasDrotmg_v2",
             ("rocblas_drotmg", CONV_MATH_FUNC, API_BLAS, HIP_UNSUPPORTED),
         ),
+        ("cublasComputeType_t", ("hipblasLtComputeType_t", CONV_MATH_FUNC, API_BLAS)),
+        ("CUBLAS_COMPUTE_32F", ("HIPBLASLT_COMPUTE_F32", CONV_MATH_FUNC, API_BLAS)),
+        ("CUBLAS_COMPUTE_64F", ("HIPBLASLT_COMPUTE_F32", CONV_MATH_FUNC, API_BLAS)),
+        ("CUBLAS_COMPUTE_32F_FAST_TF32", ("HIPBLASLT_COMPUTE_TF32_FAST", CONV_MATH_FUNC, API_BLAS)),
```

Comment:
These mappings are going from CUBLAS to HIPBLASLT (instead of HIPBLAS). Just wanted to confirm if this is intentional and okay. I didn't find any of these mappings in Jeff's PR (https://github.com/pytorch/pytorch/pull/105881/files#diff-85bd10d67a85149584e7d7a8cba533241f7ad14450e5d54ffec23da34032429a), so would like to double-check cc @jeffdaily